src/DynamoWPFCLI/Program.cs

```diff
@@ -29,11 +30,12 @@ internal static void Main(string[] args)
                 useConsole = !cmdLineArgs.NoConsole;
                 var locale = StartupUtils.SetLocale(cmdLineArgs);
 
-                if (cmdLineArgs.DisableAnalytics)
+                if (cmdLineArgs.DisableAnalytics || cmdLineArgs.NoNetworkMode)
```

Comment:
see these checks repeated in each exe, any chance of centralizing them so we can't forget to make these checks in new apps?